src/common/valgen.cpp

```diff
@@ -499,6 +521,25 @@ bool wxGenericValidator::TransferFromWindow()
             *m_pBool = pControl->GetValue() ;
             return true;
         }
+        else if (m_pInt)
+        {
+            *m_pInt = -1;
```

Comment:
Very minor, but I think this might be a tad more readable:
```suggestion
            *m_pInt = wxNOT_FOUND;
```